clients/admin-ui/src/features/data-discovery-and-detection/action-center/MonitorResultDescription.tsx

```diff
@@ -28,6 +28,7 @@ export const MonitorResultDescription = ({
     return Object.entries(updates)
       .filter(
         ([key, count]) =>
+          count != null &&
```

Comment:
Always use strict checking in Typescript. If this is intentionally allowing for undefined it would be better to strict check that as well for code clarity
```suggestion
          count !== null &&
```
for example:
`undefined != null` -> false
`undefined !== null` -> true

Reply:
Fixed!